app/v2/controllers/CreateAmendForeignPropertyAnnualSubmissionController.scala

```diff
@@ -23,47 +23,48 @@ import play.api.mvc.{ Action, ControllerComponents }
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.audit.http.connector.AuditResult
 import utils.{ IdGenerator, Logging }
-import v2.controllers.requestParsers.AmendForeignPropertyAnnualSubmissionRequestParser
+import v2.controllers.requestParsers.CreateAmendForeignPropertyAnnualSubmissionRequestParser
```

Comment:
I think some of these need reformatting, can a 'reformat code' be done on `app`, `it` and `test`?

Reply:
This is done now, it's a lot of files changed!